packages/marketing-modal/src/MarketingModal.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import Button from '@leafygreen-ui/button';
+import { Link } from '@leafygreen-ui/typography';
+import Modal from '@leafygreen-ui/modal';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const titleStyle = css`
+  color: ${uiColors.gray.dark2};
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 25px;
+
+  margin-bottom: 10px;
+`;
+
+const baseModalStyle = css`
+  width: 600px;
+  padding: initial;
+  overflow: scroll;
+`;
+
+const baseCoverStyle = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  & > * {
+    display: block;
+  }
+`;
+
+const defaultCoverStyle = css`
+  padding-top: 20px;
+  padding-bottom: 8px;
+`;
+
+const coverCoverStyle = css`
+  padding-bottom: 24px;
+
+  & > * {
+    width: 100%;
+  }
+`;
+
+const contentStyle = css`
+  color: ${uiColors.gray.dark1};
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 24px;
+  text-align: center;
+
+  padding: 0 92px;
+  padding-bottom: 24px;
+`;
+
+const footerContentStyle = css`
+  line-height: 24px;
+  padding-bottom: 50px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  & > * {
+    padding: 4px;
+  }
+`;
+
+interface MarketingModalProps {
+  title: string;
+  cover: JSX.IntrinsicElements['img'];
```

Comment:
Should we only allow for image tags? I can imagine wanting to use CSS background images, an inline SVG, or a canvas element here.

Reply:
I chose just image tags as a starting point since I hadn't come up with a one-size-fits-all styling solution yet. I can work at it more if you have a sense that this will be a blocker for most consumers to begin using the component.